apiserver/root.go

```diff
@@ -928,7 +935,11 @@ func (ctx *facadeContext) ModelExporter(modelUUID model.UUID, backend facade.Leg
 	domainServices := ctx.DomainServicesForModel(modelUUID)
 	coordinator := modelmigration.NewCoordinator(logger)
 
-	exporter := domainmodelmigration.NewExporter(coordinator, logger, clock)
+	objectStoreGetter := modelObjectStore(func(iCtx context.Context) (objectstore.ObjectStore, error) {
```

Comment:
nit: 
```suggestion
	objectStoreGetter := modelObjectStore(func(ctx context.Context) (objectstore.ObjectStore, error) {
```

Reply:
It cannot be named `ctx` because that's the variable name of the receiver of the outer method. I renamed it to `stdCtx` though, which is what we do elsewhere. `iCtx` was just a poorly chosen name. Thanks!